apps/examples/task_manager_sample/README.md

```diff
@@ -0,0 +1,24 @@
+# examples/task\_manager\_sample
+  This is the Task Manager 'qualification' suite. It attempts to exercise how to control tasks via the task manager
+. Its coverage is not very extensive as of this writing, but it is used to qualify each TizenRT release.
+
+
+## Sample Scenario:
+1. The task\_manager\_sample\_main in the task\_manager\_sample.c is the entry point of the task manager sample
+2. First, the task\_manager\_sample\_main registers an action\_manager and a user\_app through the task manager.
+3. After registering the action\_manager and the user\_app, the task\_manager\_sample\_main requests to start the action\_manager to the task manager.
+4. When the action\_manager is started, the action\_manager registers an alarm action, a LED on action, and a LED off action through the task manager.
+5. After register procedure of the action\_manager, the action\_manager requests to start the LED on action and the LED off action sequentially through the task manager.
```

Comment:
```through the task manager``` is not important. This is a sample app of task manager. That sentence is always right.
Let's give an information which API is used to do that work.
i.e. ```using task_manager_start().```